remove-node.yml

```diff
@@ -8,32 +8,32 @@
 - hosts: "{{ node | default('etcd:k8s_cluster:calico_rr') }}"
   gather_facts: no
   environment: "{{ proxy_disable_env }}"
-  vars_prompt:
-    name: "delete_nodes_confirmation"
-    prompt: "Are you sure you want to delete nodes state? Type 'yes' to delete nodes."
-    default: "no"
-    private: no
+  tasks:
+    - name: Confirm Execution
+      pause:
+        prompt: "Are you sure you want to delete nodes state? Type 'yes' to delete nodes."
+      register: pause_result
+      run_once: True
+      when:
+        - not (skip_confirmation | default(false) | bool)
 
-  pre_tasks:
-    - name: check confirmation
+    - name: Fail if user does not confirm deletion
       fail:
         msg: "Delete nodes confirmation failed"
-      when: delete_nodes_confirmation != "yes"
+      when: pause_result.user_input | default('yes') != 'yes'
 
 - hosts: kube_control_plane[0]
   gather_facts: no
   environment: "{{ proxy_disable_env }}"
   roles:
     - { role: kubespray-defaults }
-    - { role: bootstrap-os, tags: bootstrap-os }
```

Comment:
bootstrap-os role has been implemented since https://github.com/kubernetes-sigs/kubespray/pull/6154
I am not sure what flatcar means on the pull request.

/cc @chifu1234

Reply:
Many people are facing issues related to this part, and I cannot find clear reason of this additional part.
So it is fine to go forward with this change and let's see what happens.

/lgtm